test/docs/heuristic/bad-dates.html

```diff
@@ -11,6 +11,7 @@ <h2 class='foo'>Has an h2.foo</h2>
       <h2 id="abstract">Abstract</h2>
       <p>Hi! This is an abstract</p>
       <h2 id="sotd">Status of This Document</h2>
+      <!-- fail heuristic.date-format, candidateDates:  ["13/Jan/1998","february 2 1998"], note that '2014-11-02' is not a candidate -->
```

Comment:
can you please add the reference string causing the issue into the test "HSHQDC-16-R00012-H-SB2016-1-002, 70RSAT20T00000010" ?